public/app/features/alerting/unified/components/MenuItemPauseRule.tsx

```diff
@@ -33,30 +39,36 @@ const MenuItemPauseRule = ({ rule, onPauseChange }: Props) => {
       return;
     }
     const ruleUid = rule.rulerRule.grafana_alert.uid;
+    const targetGroup = await getGroup({
+      rulerConfig: grafanaRulerConfig,
+      namespace: rule.namespace.uid || rule.rulerRule.grafana_alert.namespace_uid,
+      group: rule.group.name,
+    });
 
     // Parse the rules into correct format for API
-    const modifiedRules = group.rules.map((groupRule) => {
-      if (!(isGrafanaRulerRule(groupRule.rulerRule) && groupRule.rulerRule.grafana_alert.uid === ruleUid)) {
-        return groupRule.rulerRule!;
-      }
-
-      return produce(groupRule.rulerRule, (updatedGroupRule) => {
-        updatedGroupRule.grafana_alert.is_paused = newIsPaused;
-      });
-    });
+    const modifiedRules =
+      targetGroup.data?.rules.map((groupRule: RulerRuleDTO) => {
```

Comment:
We don't need to specify the type here, type inference will do it correctly for us :)

Reply:
true true... sometimes i add the type to be able to navigate to it while i'm coding. I forgot to remove it